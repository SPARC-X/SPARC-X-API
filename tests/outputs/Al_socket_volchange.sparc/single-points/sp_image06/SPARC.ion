# Ion File Generated by SPARC ASE Calculator
# 
# ASE-SORT:
# 0 1 2 3
# END ASE-SORT

ATOM_TYPE: Al
N_TYPE_ATOM: 4
PSEUDO_POT: 13_Al_3_1.9_1.9_pbe_n_v1.0.psp8
COORD:
-0.05832938958648 0.13638415826853 0.04075054176378
-0.16820468848862 3.32319434847749 3.95928218142642
3.99928278584834 -0.28326093157844 4.09511175831219
3.70800271465139 4.28069601250493 0.11259064365630

