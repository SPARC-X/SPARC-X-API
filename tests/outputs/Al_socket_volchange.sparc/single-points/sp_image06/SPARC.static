***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of Al:
      0.9923016377       0.0180000969       0.0053782911
      0.9778002026       0.4385980086       0.5225494203
      0.5278287340       0.9626149820       0.5404763230
      0.4893853431       0.5649698903       0.0148598086
Total free energy (Ha): -9.035924426688817E+00
Atomic forces (Ha/Bohr):
  5.5845432699E-03   2.7105682219E-03   1.0197450819E-02
  8.5972146543E-03   5.2504361549E-03  -3.2509737736E-03
 -1.5552104596E-02   1.5480318661E-03  -1.3859174812E-02
  1.3703466717E-03  -9.5090362429E-03   6.9126977664E-03
Stress (GPa): 
  1.7759555830E+01  -4.6546546866E-01  -6.3891690648E-01 
 -4.6546546866E-01  -4.4874740031E+01   1.8066076363E+00 
 -6.3891690648E-01   1.8066076363E+00  -2.5353486003E+01
