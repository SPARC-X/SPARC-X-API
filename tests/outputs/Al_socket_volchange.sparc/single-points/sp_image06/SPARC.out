***************************************************************************
*                       SPARC (version Oct 31, 2023)                      *
*   Copyright (c) 2020 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Thu Jan 18 08:03:31 2024                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
LATVEC_SCALE: 1 1 1 
LATVEC:
7.576856901543160 0.000000000000000 0.000000000000000 
0.000000000000000 7.576856901543160 0.000000000000000 
0.000000000000000 0.000000000000000 7.576856901543160 
FD_GRID: 15 15 15
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.000950043469
EXCHANGE_CORRELATION: GGA_PBE
NSTATES: 12
CHEB_DEGREE: 17
CHEFSI_BOUND_FLAG: 0
CALC_STRESS: 1
MAXIT_SCF: 100
MINIT_SCF: 2
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-03
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-05
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-06
MIXING_VARIABLE: density
MIXING_PRECOND: kerker
TOL_PRECOND: 2.55E-04
PRECOND_KERKER_KTF: 1
PRECOND_KERKER_THRESH: 0
MIXING_PARAMETER: 1
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
NUM_CHEFSI: 1
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
PRINT_ENERGY_DENSITY: 0
OUTPUT_FILE: SPARC
***************************************************************************
                                Cell                                       
***************************************************************************
Lattice vectors (Bohr):
7.576856901543160 0.000000000000000 0.000000000000000 
0.000000000000000 7.576856901543160 0.000000000000000 
0.000000000000000 0.000000000000000 7.576856901543160 
Volume: 4.3497796325E+02 (Bohr^3)
Density: 2.4811867064E-01 (amu/Bohr^3), 2.7803863277E+00 (g/cc)
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 2
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 1 1 2
EIG_SERIAL_MAXNS: 1500
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  2
Mesh spacing                       :  0.505124 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  SPARC.out
Total number of atom types         :  1
Total number of atoms              :  4
Total number of electrons          :  12
Atom type 1  (valence electrons)   :  Al 3
Pseudopotential                    :  13_Al_3_1.9_1.9_pbe_n_v1.0.psp8
Atomic mass                        :  26.9815385
Pseudocharge radii of atom type 1  :  7.07 7.07 7.07 (x, y, z dir)
Number of atoms of type 1          :  4
Estimated total memory usage       :  3.01 MB
Estimated memory per processor     :  1.50 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.2605458078E+00        2.146E-01        0.020
2            -2.2621573480E+00        2.408E-01        0.007
3            -2.2594786828E+00        1.180E-01        0.007
4            -2.2591357769E+00        7.120E-02        0.007
5            -2.2590005980E+00        1.797E-02        0.006
6            -2.2589859989E+00        9.970E-03        0.007
7            -2.2589833120E+00        5.697E-03        0.006
8            -2.2589804247E+00        1.799E-03        0.006
9            -2.2589817326E+00        1.118E-03        0.006
10           -2.2589811067E+00        4.398E-04        0.006
Total number of SCF: 10    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.2589811067E+00 (Ha/atom)
Total free energy                  : -9.0359244267E+00 (Ha)
Band structure energy              :  6.0207687273E-01 (Ha)
Exchange correlation energy        : -4.4111801542E+00 (Ha)
Self and correction energy         : -1.2498686849E+01 (Ha)
-Entropy*kb*T                      : -2.2238953722E-03 (Ha)
Fermi level                        :  1.3889647577E-01 (Ha)
RMS force                          :  1.3812018571E-02 (Ha/Bohr)
Maximum force                      :  2.0888778961E-02 (Ha/Bohr)
Time for force calculation         :  0.016 (sec)
Pressure                           :  1.7489556735E+01 (GPa)
Maximum stress                     :  4.4874740031E+01 (GPa)
Time for stress calculation        :  0.025 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  0.204 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*               List of contributors: See the documentation               *
*         Citation: See README.md or the documentation for details        *
*  Acknowledgements: U.S. DOE SC (DE-SC0019410), U.S. DOE NNSA (ASC)      *
*      {Preliminary developments: U.S. NSF (1333500,1663244,1553212)}     *
***************************************************************************
                                                                           
